runtimes/runtimes/operational-telemetry/operational-telemetry.ts

```diff
@@ -0,0 +1,22 @@
+export type OperationalMetric = {
+    name: string
+    value: number
+    timestamp: number
+    atrributes: { string: any }[]
+    metrics: {}
+    server: {
+        name: string
+        version: string
+    }
+    clientInfo: {
```

Comment:
I think we will need all the fields from `clientInfo` that we get from destination.